static/js/marker.js

```diff
@@ -39,7 +39,15 @@ var MarkerView = Backbone.View.extend({
 
 		this.$el.width(400);
 		this.$el.find(".title").text(TYPES_MAP[this.model.get("title")]);
-		this.$el.find(".description").text(this.model.get("description"));
+		this.$el.find(".roadType").text(this.model.get("roadType"));
```

Comment:
The Integer pulling is done. I added load_extra_data values to Marker (main street, secondary street and junction) - these are getting a Text column due to their nature..

Now about the JS localization I would need some guidance. 
localization should be js and the function should be a js function in marker.js?